documentation/spring-integration/index.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: docs
+title: Documentation — Spring integration
+submenu: spring-integration
+---
+
+# Spring integration
+
+<h2 name="introduction">Introduction</h2>
+
+One of our main approach is Javers should be easy to use, with this in mind we prepared integration for
+applications based on Spring framework. Integration is based on spring-context and spring-aop. As a customer only one thing you
+have to do is to annotate a methods that modify your data, everything else is done by Javers Spring Integration.
+
+<h2 name="usage">Usage</h2>
+
+There are few steps you need to go through:
+
+<h3>Enable AspectJ Auto Proxy</h3>
+
+First thing you have to do is enable AspectJ Auto Proxy. Javers add aspects that commit changes after invoking annotated
+methods, this is not possible without enabled AspectJ. For to more info go to Spring documentation:
+
+[docs.spring.io](http://docs.spring.io/spring/docs/current/spring-framework-reference/html/aop.html#aop-aspectj-support)
+
+<h3>Add AuthorProvider bean definition to application context</h3>
+
+Next step is providing implementation of [AuthorProvider](https://github.com/javers/javers/blob/664a2d2a3f8eec57f5f5647bcd23aea25e3b5f4f/javers-spring/src/main/java/org/javers/spring/AuthorProvider.java) interface:
```

Comment:
znowu, brakuje kontekstu, po co mi ten author provider? trzeba opisać, że chodzi o to że transakcje wykonywane na danych powinny być podpisane przez użytkownika aplikacji, i że użytkownika bieżącej transakcji/sesji powinien dostarczyć ten bean. Można zasugerować implementację, np w oparciu o sesje SpringSecurity czy coś w tym stylu